build.gradle

```diff
@@ -298,7 +298,11 @@ dependencies {
 
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation'
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web-services'
+  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-cache'
+  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-redis'
   implementation group: 'org.springframework', name: 'spring-context'
+  implementation 'org.springframework.integration:spring-integration-redis:6.2.1'
+  testImplementation 'it.ozimov:embedded-redis:0.7.2'
```

Comment:
Should we not be trying to keep the dependencies together in some similar sort of style
- should these be in the `group, name, version` style?
- keep the `testImplementation` stuff together also

Reply:
Done